tox.ini

```diff
@@ -11,6 +11,11 @@ max-line-length = 120
 show-source = True
 builtins = get_ipython
 
+[flake8-rst]
+filename = *.rst *.py
+max-line-length = 120
+ignore = E501,F821,E402
```

Comment:
1. You no needed `E501`, because you specified `max-line-length`.
2. Why `E402`, is it produce problems with docstring?
3. `F821` shouldn't be here too, but this is for future (when we fix all docstrings)